src/quickgui/plugin/qgsquickphotopanel.qml

```diff
@@ -87,28 +88,27 @@ Drawer {
           photoPreview.source = preview
         }
       }
+
+      focus {
+          focusMode: Camera.FocusMacro
+          focusPointMode: Camera.FocusPointCustom
+          customFocusPoint: Qt.point(0.2, 0.2) // Focus relative to top-left corner
+      }
```

Comment:
Is this here on purpose or a leftover from an experiment?

Reply:
Added a fix in a recent commit. Focus hasn't been working in the previous version.